server/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponseTests.java

```diff
@@ -64,16 +61,4 @@ public void testToXContent() {
             {"acknowledged":true,"shards_acknowledged":false,"index":"index_name"}""", output);
     }
 
-    public void testToAndFromXContentIndexNull() throws IOException {
```

Comment:
Test is obsolete. 
Node-to-node serialization doesn't allow a null index, while xcontent serialization currently does.